src/Response/Struct/ItemOperatingHours.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\InpostBundle\Response\Struct;
+
+class ItemOperatingHours
+{
+    public array $sunday = [];
+    public array $monday = [];
+    public array $tuesday = [];
+    public array $wednesday = [];
+    public array $thursday = [];
+    public array $friday = [];
+    public array $saturday = [];
+
+    public static function fromArray(array $operatingHours): self
+    {
+        $self = new self();
+
+        if (!isset($operatingHours['customer'])) {
+            return $self;
+        }
+
+        if (!is_array($operatingHours['customer'])) {
+            return $self;
+        }
```

Comment:
```suggestion
        if (!empty($operatingHours['customer'] ?? [])) {
            return $self;
        }
```

#whatever

Reply:
In my opinion, the suggested code does not check whether a variable is an array